python/lsst/ts/lasertracker/mock/mock_utils.py

```diff
@@ -67,7 +77,7 @@ def get_random_initial_position() -> dict[str, MockT2SAPointGroup]:
         m1m3=MockT2SAPointGroup(
             origin=CartesianCoordinate(*np.random.normal(0.0, 1e-3, 3)),
             rotation=BodyRotation(*np.random.normal(0.0, 6e-3, 3)),
-            radius=8.40,
```

Comment:
Why are you changing this?

Reply:
Because 8.40 is the diameter of the telescope, not the radius, no?